### PR TITLE
Hides previous question when correct

### PR DESCRIPTION
**Pull request by rapheh**
hi

### CODE REVIEW

**Review by kateherz — APPROVED** [review]
looks good!

File: scripts/main.js [hunk 1]
```diff
@@ -77,7 +77,10 @@ var MyScene = new Phaser.Class({
             var trueFalse = this.checkAnswer(name.value);
             if (trueFalse == true) {
                 this.correctText.setText("Correct!");
-                this.currentQuestionIndex = null;           
+                this.currentQuestionIndex = null; 
+                this.text.destroy();
```
Comment (akundla) [inline scripts/main.js:81]: should be called questionText

File: scripts/main.js [hunk 2]
```diff
@@ -77,7 +77,10 @@ var MyScene = new Phaser.Class({
...
+                this.r1.destroy();
```
Comment (akundla) [inline scripts/main.js:82]: should be called questionRectangle or rectangleBackground or questionBackground

**Review by akundla — COMMENTED** [review]
needs a few var renames

**Pull request merged**